pkg/cmd/server/start/start_master.go

```diff
@@ -429,8 +429,10 @@ func StartAPI(oc *origin.MasterConfig, kc *kubernetes.MasterConfig) error {
 	}
 
 	// verify we can connect to etcd with the provided config
```

Comment:
add a TODO to switch this to testing using the `oc.EtcdHelper`... the fewer things dig down into the config for EtcdClientInfo, the better